cpp/ql/src/Likely Bugs/NestedLoopSameVar.qhelp

```diff
@@ -26,7 +26,7 @@ outer loop. </p>
   Tutorialspoint - The C++ Programming Language: <a href="http://www.tutorialspoint.com/cplusplus/cpp_nested_loops.htm">C++ nested loops</a>
 </li>
 <li>
-  MSDN Library: <a href="http://msdn.microsoft.com/en-us/library/8y82wx12%28v=VS.80%29.aspx">Nested Control Structures</a>
+  MSDN Library: <a href="https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/language-features/control-flow/nested-control-structures">Nested Control Structures</a>
```

Comment:
Strange that this page is about Visual Basic, in the C/C++ query help.  The points on the page apply perfectly well to C/C++, but I can't help but think we could do better.

Reply:
Perhaps https://www.programiz.com/cpp-programming/nested-loops ?

Reply:
I agree that it's a strange choice. I looked on [Wayback Machine](https://web.archive.org/web/20110407080413/http://msdn.microsoft.com/en-us/library/8y82wx12(v=VS.80).aspx) at the old link, and that apparently also referenced Visual Basic. If we don't mind linking to tutorials then I think your link explains the concepts equally well (and with the bonus of being written in C++!)

Reply:
Actually... after looking at the qhelp file again I realized that we already have this one linked:
```html
Tutorialspoint - The C++ Programming Language: <a href="http://www.tutorialspoint.com/cplusplus/cpp_nested_loops.htm">C++ nested loops</a>
```
so maybe adding https://www.programiz.com/cpp-programming/nested-loops is somewhat redundant. What if we just remove that link altogether?

Reply:
I'm happy with it just removed.

Reply:
Removed in 9db66a1b9451c08122eabca386d0b1b40f317021.